internal/x/exec/runner.go

```diff
@@ -340,6 +341,7 @@ func (r *runner) Lint(args []string) error {
 }
 
 func (r *runner) lint(meta *meta) error {
+	r.logger.Debug("calling LintRunner")
```

Comment:
just a nit: since this is exposed to cli users who don't know what a `lintRunner` is, should we just have this as `"calling lint"`?

Reply:
Debug lines expose a lot of internals on purpose, see the logging lines in the file and settings packages for example.

Reply:
👍